category.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * The template for displaying Taxonomy pages.
+ *
+ * Used to display taxonomy-type pages
+ *
+ * Learn more: http://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package  WordPress
+ * @subpackage  Timber
+ */
+
+use Timber\Timber;
+
+$templates = [ 'page_type.twig', 'index.twig' ];
```

Comment:
This is using the `page_type` template, I guess the `category` pages should look more or less the same, but maybe just copying the `page_type` template to a new one could be better for adapting to tiny differences.

Reply:
...OR we could change the `page_type`, and `tease-page-type` files into something more generic, like `taxonomy` and `tease-taxonomy`, given that a `category` is a `taxonomy` too... I'll wait for other opinions on this.

Reply:
The correct way (throughout the project) would be that all common elements are defined on their own. (For example `article_excerpt.twig`), and then included in other common elements (for example articles_list.twig), and then included in any template that needs them. 
(For example inside page_type.twig, and in categories.twig etc). 
Like that we would have consistency (changing the design in one place should change in all). 

For now, if they have common parts, then 
1) find a name describing the common part, 
2) create a template for it
3) create a category.twig
4) Inside the category.twig call the common part
5) Inside the page_type.twig call the common part
6) change the above line 15 to call the category.twig